examples/typescript-backbone/js/app.ts

```diff
@@ -319,6 +369,12 @@ class AppView extends Backbone.View {
 				completed: completed,
 				remaining: remaining
 			}));
+
+			this.$('.filters li a')
+				.removeClass('selected')
+				.filter('[href="#/' + (taskFilter || '') + '"]')
```

Comment:
:( I hate using the dom like this... but I know it is how other examples work :(